ReleaseNotes.md

```diff
@@ -11,6 +11,23 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod\<mmddyyyy\>), which indicate the agent build dates (not release dates)
 
+### 11/29/2022 -
+##### Version microsoft/oms:ciprod11292022-6a507f9a Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11292022-6a507f9a (linux)
+##### Version microsoft/oms:win-ciprod11292022-6a507f9a Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod11292022-6a507f9a (windows)
+##### Code change log
+
+- Windows Agent
+  - Use ghrelease as gcc download path
+  - Move conf update before fluent-bit start
+  - Make containerd and secure port as default for windows same as linux
+- Common (Linux & Windows Agent)
+  - Add telegraf config options to configmap
```

Comment:
Rephrase as "Add FluentBit Telegraf Plugin buffer and chunk config options to configmap"

Reply:
updated